packages/access-client/src/agent-use-cases.js

```diff
@@ -234,7 +233,7 @@ export async function waitForAuthorizationByPolling(access, opts = {}) {
  * @param {AbortSignal} [opts.signal]
  * @param {boolean} [opts.dontAddProofs] - whether to skip adding proofs to the agent
  * @param {Iterable<{ can: Ucanto.Ability }>} [opts.capabilities]
- * @param {AuthorizationWaiter} [opts.expectAuthorization] - function that will resolve once account has confirmed the authorization request
+ * @param {import('./types.js').AuthorizationWaiter<any>} [opts.expectAuthorization] - function that will resolve once account has confirmed the authorization request
```

Comment:
* I think we shouldn't make this change because we shouldn't make AuthorizationWaiter generic
* and because it uses `any`. Whenever you are tempted to use `any`, you probaby want `unknown` or something else
  * I would ahve thought we had [this rule](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/no-explicit-any.md) enabled that might complain. It's a good rule.